### ISSUE TITLE
stb_truetype.h: stbtt_pack_range using glyph indices instead of codepoints?

### ISSUE THREAD

**Issue by pixelblender** [nothings/stb#405]
I tried to use HarfBuzz with stb_truetype.h. The issue is that while iterating characters HarfBuzz gives glyph indices but stbtt_pack_range only support codepoints. Is there a way to pack font and generate atlas with stbtt_pack_range using glyph indices?

**Comment by nothings** [comment]
Nope, I wasn't aware you'd have to do it that way. Sounds like it's worth adding.
